irohad/ametsuchi/impl/postgres_query_executor.hpp

```diff
@@ -146,20 +146,60 @@ namespace iroha {
        * Execute query which returns list of transactions
        * uses pagination
        * @param query - query object
+       * @param qry_checker - fallback checker of the query, needed in some
```

Comment:
some unclear cases -> if paging hash is not specified and 0 transactions are returned